docs/UserGuide.md

```diff
@@ -373,6 +373,39 @@ Examples:
 * `eremovev vid/1 eid/1` 
   * removes the volunteer with id 1 from the event with id 1.
 
+### Undo a Command: `undo`
+
+Undoes an undo-able command.
+
+Format: `undo`
+
+Restrictions:
+* Undo-able commands include:
+  * vcreate
+  * vdelete
+  * vedit
+  * vclear
+  * ecreate
+  * edelete
+  * eedit
+  * eaddv
+  * eremovev
+  * eaddm
+* Only commands that changed the current history stack can be undone (iVolunteer does not remember commands that were executed in the previous run of the app).
+
+### Redo a Command: `redo`
+
+Redoes the undo command. 
+
+Format: `redo`
```

Comment:
can give an example of how to use? (im joking)

Reply:
hi joking!